### PR TITLE
Add 3.2.4 announcement news, release note and download link

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
- Apache Download: https://downloads.apache.org/spark/spark-3.2.4/
- Maven Central: https://repo1.maven.org/maven2/org/apache/spark/spark-core_2.12/3.2.4/
- PyPI: https://pypi.org/project/pyspark/3.2.4/
- DockerHub: https://hub.docker.com/r/apache/spark/tags?page=1&name=v3.2.4

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Could you review this, @viirya , @sunchao , @huaxingao , @gengliangwang ?

File: releases/_posts/2023-04-13-spark-release-3-2-4.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - [[SPARK-42697]](https://issues.apache.org/jira/browse/SPARK-42697): /api/v1/applications return 0 for duration
+  - [[SPARK-42747]](https://issues.apache.org/jira/browse/SPARK-42747): Fix incorrect internal status of LoR and AFT
+  - [[SPARK-42785]](https://issues.apache.org/jira/browse/SPARK-42785): When spark submit without --deploy-mode, will face NPE in Kubernetes Case
+  - [[SPARK-42906]](https://issues.apache.org/jira/browse/SPARK-42906): Replace a starting digit with `x` in resource name prefix
+  - [[SPARK-42967]](https://issues.apache.org/jira/browse/SPARK-42967): Fix SparkListenerTaskStart.stageAttemptId when a task is started after the stage is cancelled
+  - [[SPARK-43004]](https://issues.apache.org/jira/browse/SPARK-43004): vendor==vendor typo in ResourceRequest.equals()
+
+### Dependency Changes
+
+While being a maintence release we did still upgrade some dependencies in this release they are:
```
Comment (viirya) [inline releases/_posts/2023-04-13-spark-release-3-2-4.md]: typo?
```suggestion
While being a maintenance release we did still upgrade some dependencies in this release they are:
```

**Review by viirya — APPROVED** [review]
Found a typo, otherwise looks good.

File: releases/_posts/2023-04-13-spark-release-3-2-4.md (hunk 1 above)
Comment (dongjoon-hyun) [reply releases/_posts/2023-04-13-spark-release-3-2-4.md]: Oops. Thanks!

**Comment by dongjoon-hyun** [comment]
Thank you!

**Comment by huaxingao** [comment]
Late LGTM

**Comment by dongjoon-hyun** [comment]
Thank you, @huaxingao .

**Comment by viirya** [comment]
@dongjoon-hyun Seems html files is not updated together? The CI failed due to that.

```
--- a/site/releases/spark-release-3-2-4.html
+++ b/site/releases/spark-release-3-2-4.html
@@ -162,7 +162,7 @@
 
 <h3 id="dependency-changes">Dependency Changes</h3>
 
-<p>While being a maintence release we did still upgrade some dependencies in this release they are:</p>
+<p>While being a maintenance release we did still upgrade some dependencies in this release they are:</p>
 ```

**Comment by dongjoon-hyun** [comment]
Ah, got it.

**Comment by dongjoon-hyun** [comment]
I fixed the typo direclty, @viirya ~
- https://github.com/apache/spark-website/commit/35b8f45bdffcc9b0d9ff52c849cbdb01d17ef95f

**Comment by viirya** [comment]
Thank you @dongjoon-hyun !

**Comment by gengliangwang** [comment]
Late LGTM. I will update the Algolia Crawler

**Comment by dongjoon-hyun** [comment]
Thank you so much always, @gengliangwang ! Yes, please~
> Late LGTM. I will update the Algolia Crawler

**Pull request merged**